pkgs/development/libraries/libhugetlbfs/default.nix

```diff
@@ -40,10 +40,14 @@ stdenv.mkDerivation rec {
   installTargets = [ "install" "install-docs" ];
 
   meta = with lib; {
-    broken = (stdenv.isLinux && stdenv.isAarch64);
```

Comment:
This should still be marked broken, and aarch64 checked for in nvme-cli, IMO.  There's a difference in semantics between `broken` and `platforms` that comes down to "it's supposed to work, but doesn't" vs "it isn't supposed to work".

Reply:
It would be nice to have a way to do that without duplicating the check. Maybe `lib.meta.brokenOn`?

Reply:
Maybe — it's come up before, but people were worried about things flip-flopping on and off unexpectedly too much if we had implicit behaviour changes depending on `broken`.

Reply:
This was marked as broken in some ZHF commit 11ee22d797016be02743a9973fe09e0362bcb91b
But it actually works now and is also supported by the upstream project.

Reply:
Then should we just drop it?

Reply:
https://github.com/NixOS/nixpkgs/pull/264539